app/gateway/2.6.x/install-and-run/docker.md

```diff
@@ -289,34 +271,45 @@ The steps involved in starting Kong in [DB-less mode](/gateway/{{page.kong_versi
     described in the [Declarative Configuration format] guide. Add whatever core
     entities (Services, Routes, Plugins, Consumers, etc) you need to this file.
 
+    For example, a simple file with a Service and a Route
+    could look something like this:
+
+    ```yaml
+    _format_version: "1.1"
+    _transform: true
+
+    services:
+    - host: mockbin.org
+      name: example_service
+      port: 80
+      protocol: http
+      routes:
+      - name: example_route
+        paths:
+        - /mock
+        strip_path: true
+    ```
+
     This guide assumes you named the file `kong.yml`.
 
-1.  Save your declarative configuration file inside the `MountPoint` path
-mentioned in the previous step. In the case of this
-guide, that would be `/var/lib/docker/volumes/kong-vol/_data/kong.yml`.
+1.  Save your declarative configuration locally, and note the filepath.
+For this example, we're using `/kong/declarative/kong.yml`.
```

Comment:
You don't need to note the filepath here. The `$(pwd)` uses the current directory, and `/kong/declarative` is the path inside the container that we mount to.

What's important is that they run the `docker run` command in the same directory as the `kong.yml` that they just created